doc/man3/flux_kvs_lookup.adoc

```diff
@@ -96,8 +96,7 @@ lookup.
 
 `flux_kvs_lookup_cancel()` cancels a stream of lookup responses
 requested with FLUX_KVS_WATCH or a waiting lookup response with
-FLUX_KVS_WAITCREATE (see below). The future will be fulfilled with an
-ENODATA error once the cancel request is received and processed.
+FLUX_KVS_WAITCREATE.  See `FLAGS` below for additional information.
```

Comment:
FLAGS as a section heading should not be rendered monospace so the backticks are prob not the right choice.  I looked at other man pages and when we reference sections, we have just listed the bare section name in caps.